packages/documentation/src/components/Header.jsx

```diff
@@ -1,7 +1,63 @@
+/* eslint-disable react/jsx-key */
 import React from 'react';
-import { Link } from 'gatsby';
+import { Link, graphql } from 'gatsby';
 import sidebarData from '../sidebar';
 import MobileNav from './MobileNav';
+import { Index } from 'elasticlunr';
+// Graphql query used to retrieve the serialized search index.
+export const query = graphql`
```

Comment:
the documentation doesn't say where this comes from or what to do with it. the example uses [this](https://github.com/andrew-codes/gatsby-plugin-elasticlunr-search-demo/blob/6f662e66ce0ad48f1a0bc4bf6579a618958deb29/src/templates/default.js#L1)